### DISCUSSION TITLE
Change Game Page Recent Players to Active Players

### DISCUSSION THREAD

**Discussion by timenoe** [RetroAchievements/RAWeb#1220] [Ideas]
My recommendation is to change the "Recent Players" section on a game page to "Active Players" (similar to the section on the home page).

This will remove outdated entries from recycled game pages. Below is an example of two Recent Player entries from 2018 on the game page for Magic Castle (PlayStation):

![image](https://user-images.githubusercontent.com/62894028/200946015-9a74319e-3276-4e60-af42-f8059dab6ac9.png)

Often times RP is updated to fix errors, and the old RP with errors remains displayed on the game page, because the player became inactive.

**Comment by Jamiras** [comment]
Duplicate of #1092, and will be limited to the last six months per #1197

**Reply by timenoe** [reply]
Ah, missed that, I was searching for "Recent Players". That's good to hear!
